styles/layout.css

```diff
@@ -46,16 +46,52 @@ body{
     padding:32px;
 }
 
-.nav_menu{
+.navbar{
+    min-height: 70px;
     display:flex;
-    justify-content: space-evenly;
+    justify-content: center;
     align-items: center;
-    width:100%;
+    padding: 0 24px;
+}
+
+.nav-menu{
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    gap:60px;
+}
+
+.nav-menu li{
+    list-style: none;
 }
 
-.nav_menu a{
+.nav-menu li a{
     color: white;
     font-size: 1.5rem;
+    text-decoration: none;
+}
+
+.nav-link{
+    transition: 0.7s ease;
+}
+
+.nav-link:hover{
```

Comment:
Can we make an indicator to  highlight the current page ankita .. 
Example: if we are in mammals , see the attachemnt below:

![Active_page_indication](https://github.com/user-attachments/assets/b3b966d6-557c-4830-908e-436def5d036f)

something like this.